MinnetonkaGame.py

```diff
@@ -41,12 +43,44 @@ def big_room():
         game_over("They ask you to show your skill by timing you on a Minecraft speed run. " +
                   "You fail miserably as you have never played before.")
     elif answer == "2":
-        # TODO: top_of_stairs_landing()
+        top_of_stairs_landing()
         game_over("TBD")
     else:
         big_room()
 
 
+# top of stairs
+def top_of_stairs_landing():
+    # give some prompts
+    # '\n' is to print the line in a new line
+    print("\nYou are in a living room. There is a woman speaking French. Did you teleport to France?")
+    # pause for two seconds - building suspense
+    time.sleep(2)
+    print("No. That's just Emily - an American who speaks French.")
+    time.sleep(2)
+    print("You walk over to say hi and you step on a lego. Ow!")
+    time.sleep(2)
+    print("Your foot is killing you! Do you take an aspirin or have a beer?")
+    print("1) Take an aspirin.")
+    print("2) Dull the pain with beer.")
+
+    # take input()
+    answer = input(">")
+
+    if answer == "1":
+        game_over("That aspirin was tainted with cyanide. Too bad. ")
```

Comment:
Damn, That's kinda dark. :-) We may need to re-work some of these endings so the overall theme of the game is consistent. But that can come later.

Reply:
Yes I agree, I just didn't feel like building out a branch there.